### PR TITLE
Adding MapView component

### PR DESCRIPTION
**Pull request by BajalanIman**
feature/map-view is created and ready to review. 
image

### CODE REVIEW

**Comment by orYoffe** [comment]
@BajalanIman seems like you have conflicts

**Comment by orYoffe** [comment]
@BajalanIman shouldn't we add a screenshot so it looks like the site, for now?

**Comment by orYoffe** [comment]
> @BajalanIman shouldn't we add a screenshot so it looks like the site, for now?

I mean instead of the grey area

**Comment by BajalanIman** [comment]
But it is a map, it's not an image. I'm not sure whether importing the map in the future will be the same as an image...

**Comment by Konzisam** [comment]
In this case I would recommend adding the image in your path, then it reduces the work when it comes to reviewing: I dont have to download an image to replicate the code which is unnecessary extra work, better to keep the image in the project. Otherwise the components works as per requirements, Good work!

**Comment by Konzisam** [comment]
src/components/mapView/MapView.jsx

Also a small one - typo-  I guess you meant to write map-view-sample-img.png, not map-view-samle-img.png

File: src/components/mapView/MapView.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./MapView.css";
+import mapViewSamleImg from "./../../../public/map-view-samle-img.png";
```
Comment (Konzisam) [inline src/components/mapView/MapView.jsx:3]: Also a small one - typo- I guess you meant to write map-view-sample-img.png, not map-view-samle-img.png

**Review by Konzisam — APPROVED** [review]

File: src/components/mapView/MapView.jsx (hunk 1 above)
Comment (BajalanIman) [reply src/components/mapView/MapView.jsx]: Thank you for your review! Both comments have been addressed.

**Pull request merged**